website/test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+enum34==1.1.6
```

Comment:
now that the website/test-requirements are separate from the project/test-requirements, can these requirements be cleaned up?

otherwise, would it make sense to just keep them as they are in a single test-requirements file?

Reply:
I've cleaned up a bit. I think they should be kept separate, as the two parts are very different in principle.